backend/api/views.py

```
@@ -1,96 +1,75 @@
+                
+                # Log the user in for the DRF session
+                login(request, user)
+                
+                return Response({
+                    "success": True,
+                    "message": "Authenticated successfully. You can now use the API browser.",
+                })
+            except User.DoesNotExist:
+                return Response({"error": "User not found in Django database"}, status=404)
+                
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)